migrations/20170708133715_ensure_crate_id_foreign_keys_cascade/up.sql

```diff
@@ -0,0 +1,21 @@
+ALTER TABLE "crate_downloads"
+    DROP CONSTRAINT "fk_crate_downloads_crate_id",
+    ADD CONSTRAINT "fk_crate_downloads_crate_id" FOREIGN KEY (crate_id) REFERENCES crates(id) ON DELETE CASCADE;
+ALTER TABLE "crate_owners"
+    DROP CONSTRAINT "fk_crate_owners_crate_id",
+    ADD CONSTRAINT "fk_crate_owners_crate_id" FOREIGN KEY (crate_id) REFERENCES crates(id) ON DELETE CASCADE;
+ALTER TABLE "crates_categories"
+    DROP CONSTRAINT "fk_crates_categories_crate_id",
+    ADD CONSTRAINT "fk_crates_categories_crate_id" FOREIGN KEY (crate_id) REFERENCES crates(id) ON DELETE CASCADE;
```

Comment:
Ahhh I was seeing the `ON DELETE CASCADE` for the category on `crates_categories`, not the crate. 

Would love to see the down migration fixed :)